### DISCUSSION TITLE
Add Abacus to F-Droid store

### DISCUSSION THREAD

**Discussion by Huddeij** [victorbalssa/abacus#264] [Ideas]
**Is your feature request related to a problem? Please describe.**
For users of degoogled androids without play store it would be awesome to find this app in the open source store f-droid

**Describe alternatives you've considered**
Add the packaged apk to the release page for manual download.

**Comment by victorbalssa** [comment]
Universal APK available here: https://github.com/victorbalssa/abacus/releases/tag/0.9.1

**Comment by AlexKalopsia** [comment]
I agree it would be nice to have the releases on F-Droid so users can also easily update to newer versions

**Comment by Huddeij** [comment]
Missing universal apk in the release version 0.9.2
And still no F-Droid release. 
Pretty disappointed by now

**Comment by mss-cyclist** [comment]
I agree, a F-Droid version will make life easier for self hosters

**Comment by victorbalssa** [comment]
Maintaining a Self-Hosted F-Droid or submitting a React Native app to the official F-Droid repo can be _very_ time-consuming and complex in the long run.

I will attach a new `universal.apk` for each minor versions, patch versions will always be available via Expo EAS Updates.

However, If someone wants to submit a request to the official F-Droid repo or include Abacus to a Self-Hosted F-Droid server, I will be happy to help for the setup.

**Reply by DodoLeDev** [reply]
Maybe that [IzzyOnDroid](https://android.izzysoft.de/), by @IzzySoft would be the way to go?

**Reply by IzzySoft** [reply]
We'd neet the APK attached to the releases. For further criteria, see the [IzzyOnDroid App Inclusion Policy](https://izzyondroid.org/docs/general/AppInclusionPolicy/) – and if you think it matches, be welcome to open an inclusion request at [our issue tracker](https://codeberg.org/IzzyOnDroid/repodata/issues) (looks still empty as we're just moving over this last piece from our last remaining GitLab repo :see_no_evil:).

Building with Expo (we always aim for [Reproducible Builds](https://izzyondroid.org/about/security/ReproducibleBuilds/)) can be a bit tricky, though – but we already have some such apps in our builders, so it should not be impossible.

**Comment by Huddeij** [comment]
How about an in app updater like the revanced manager or aliucord?

For version 0.9.2 i couldnt find an apk to download. I guess "minor version" is the second number not the third. Am i right here?

**Comment by victorbalssa** [comment]
@Trigant

Yep, I will release next minor soon: `0.10.0`.

You should have the last patch already if you accepted the in-app update (version displayed in settings shows the build apk version, so it will stay to `0.9.1` even if you applied the last patch `0.9.3`).

**Comment by victorbalssa** [comment]
New .apk version attach to https://github.com/victorbalssa/abacus/releases/tag/0.10.2

**Comment by TickDracy** [comment]
If you don't have Play Store installed, why don't you use [Aurora Store](https://auroraoss.com/) to install Abacus?
It works pretty well for Degoogled scenarios.

Another alternative is to use [Obtainium](https://github.com/ImranR98/Obtainium) and then add this repo into it so that it can alert you whenever there is an update.

**Reply by Huddeij** [reply]
Kind of ironic that aurora and obtainium are on fdroid, isn't it?

**Reply by TickDracy** [reply]
I agree that it would be better to have Abacus on F-Droid. I just shared an alternative way to get it.
